generated/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -9373,3 +9372,129 @@ def pwr_voltage_setpoint(self):
             self._handle, self._name)
         check_for_error(error_code)
 
+
+        @deprecation.deprecated
+        @property
+        def ai_eddy_current_prox_sensitivity(self):
+            return self.ai_eddy_current_prox_probe_sensitivity
+
+        @deprecation.deprecated
+        @ai_eddy_current_prox_sensitivity.setter
+        def ai_eddy_current_prox_sensitivity(self, val):
+            self.ai_eddy_current_prox_probe_sensitivity = val
```

Comment:
Did you test these deprecated properties? None of them work. They have an extra level of indentation, which causes them to be defined inside the body of `pwr_voltage_setpoint`.

```
PS C:\dev\nidaqmx-python> poetry1.2 run python
Python 3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:58:18) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import nidaqmx
>>> help(nidaqmx._task_modules.channels.ai_channel.AIChannel.ai_strain_gage_force_read_from_chan)
Help on property:

    bool: Specifies whether the data is returned by DAQmx Read when
        set on a raw strain channel that is part of a rosette
        configuration.

>>> help(nidaqmx._task_modules.channels.ai_channel.AIChannel.ai_strain_force_read_from_chan)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object 'AIChannel' has no attribute 'ai_strain_force_read_from_chan'
```

If I fix that, there are more problems.
```
>>> import nidaqmx
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Dev\nidaqmx-python\src\nidaqmx\__init__.py", line 3, in <module>
    from nidaqmx.task import Task
  File "C:\Dev\nidaqmx-python\src\nidaqmx\task.py", line 7, in <module>
    from nidaqmx._task_modules.channels.channel import Channel
  File "C:\Dev\nidaqmx-python\src\nidaqmx\_task_modules\channels\__init__.py", line 6, in <module>
    from nidaqmx._task_modules.channels.ai_channel import AIChannel
  File "C:\Dev\nidaqmx-python\src\nidaqmx\_task_modules\channels\ai_channel.py", line 35, in <module>
    class AIChannel(Channel):
  File "C:\Dev\nidaqmx-python\src\nidaqmx\_task_modules\channels\ai_channel.py", line 9382, in AIChannel
    @ai_eddy_current_prox_sensitivity.setter
AttributeError: 'function' object has no attribute 'setter'
```

I think `@deprecated` needs to be after `@property` etc.
```
    @property
    @deprecation.deprecated(deprecated_in="0.6", current_version=__version__, details="Use ai_strain_gage_force_read_from_chan instead.")
    def ai_strain_force_read_from_chan(self):
        return self.ai_strain_gage_force_read_from_chan

    @ai_strain_force_read_from_chan.setter
    @deprecation.deprecated(deprecated_in="0.6", current_version=__version__, details="Use ai_strain_gage_force_read_from_chan instead.")
    def ai_strain_force_read_from_chan(self, val):
        self.ai_strain_gage_force_read_from_chan = val

    @ai_strain_force_read_from_chan.deleter
    @deprecation.deprecated(deprecated_in="0.6", current_version=__version__, details="Use ai_strain_gage_force_read_from_chan instead.")
    def ai_strain_force_read_from_chan(self):
        del self.ai_strain_gage_force_read_from_chan
```